layerindex/tools/fedora-fetch.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+
+# Fedora Pagure (e.g. https://src.fedoraproject.org/) fetch utility
+
+# Copyright (C) 2017, 2018 Intel Corporation
+#
+# Licensed under the MIT license, see COPYING.MIT for details
+#
+# SPDX-License-Identifier: MIT
+
+import sys
+import os
+import argparse
+import json
+import requests
+import re
+import subprocess
+
+
+def get_repos_api(url, toplevel_item=None, name_attr='name', use_link_header=True, link_item='pagination', link_next_attr='next'):
+    link_re = re.compile('<(http[^>]+)>; rel="([a-zA-Z0-9]+)"')
+    repos = None
+    while True:
+        session = requests.Session()
+        print('getting %s' % url)
+        with requests.Session() as s:
+            r = s.get(url)
+            if not r.ok:
+                print('Request failed: %d: %s' % (r.status_code, r.text))
+                sys.exit(2)
+            st = r.text
+            jdata = json.loads(st)
+            if toplevel_item:
+                repos = jdata[toplevel_item]
+            else:
+                repos = jdata
+            for repo in repos:
+                name = repo[name_attr]
+                yield repo
+
+            link_url = None
+            if use_link_header:
+                link = r.headers.get('Link', None)
+                if link:
+                    linkitems = dict([reversed(x) for x in link_re.findall(link)])
+                    link_url = linkitems.get('next', None)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears you are using the `get()` method to retrieve the value for a key from a dictionary. If the key is not present, then `get()` returns the value that is passed in its optional second parameter. If if the key is not present, then `None` is the default return value. Therefore, there is no need to explicitly specify `None` in the second parameter.

[Learn more](https://docs.python.org/3/library/stdtypes.html#dict.get)